ca-es/html-ca.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+language: html
+filename: learnhtml-ca.html
```

Comment:
Please fix this to
```
filename: html-ca.html
```

Reply:
@divayprakash Hello, thanks for the speedy review! I've changed the filename to html-ca.html, I hope it is ready to merge. Best regards, Marc.